sdk/eventhub/Azure.Messaging.EventHubs/tests/Shared/TestEnvironment.cs

```diff
@@ -36,14 +36,6 @@ public static class TestEnvironment
         private static readonly Lazy<string> s_eventHubsSecretInstance =
             new Lazy<string>(() => ReadAndVerifyEnvironmentVariable("EVENT_HUBS_SECRET"), LazyThreadSafetyMode.PublicationOnly);
 
-        /// <summary>The environment variable value for the Azure Active Directory client identifier of the service principal, lazily evaluated.</summary>
```

Comment:
Just confirming this removal is intentional;  I don't see any references and am guessing that we just missed it during the design iteration for the identity infrastructure.... but, just in case.   😄

Reply:
Yes, thank you